releases/release-1.30/release-team.md

```diff
@@ -6,7 +6,7 @@
 | Emeritus Adviser | Grace Nguyen ([@gracenng](https://github.com/gracenng) / Slack: `@Grace Nguyen`) | N/A |
 | Enhancements | Mohammad Reza Saleh ([@salehsedghpour](https://github.com/salehsedghpour) / Slack: `@Mohammad Reza Saleh`) |
 | Release Notes | Rashan Smith ([@rashansmith](https://github.com/rashansmith) / Slack: `@Rashan`) |
-| Communications | Kristin Martin ([@kcmartin](https://github.com/kcmartin) / Slack: `@kcmartin`) |
+| Communications | Kristin Martin ([@kcmartin](https://github.com/kcmartin) / Slack: `@kcmartin`) | | Amit Dsouza ([@checksumz](https://github.com/checksumz) / Slack: `@AmitD` ), Frederick Kautz ([@fkautz](https://github.com/fkautz) / Slack : `@fkautz`), Abigail McCarthy ([@a-mccarthy](https://github.com/a-mccarthy) / Slack: `@abbie`), Natali Vlatko ([@natalisucks](https://github.com/natalisucks) / Slack: `@Natali`) |
```

Comment:
```suggestion
| Communications | Kristin Martin ([@kcmartin](https://github.com/kcmartin) / Slack: `@kcmartin`) |  Amit Dsouza ([@checksumz](https://github.com/checksumz) / Slack: `@AmitD` ), Frederick Kautz ([@fkautz](https://github.com/fkautz) / Slack : `@fkautz`), Abigail McCarthy ([@a-mccarthy](https://github.com/a-mccarthy) / Slack: `@abbie`), Natali Vlatko ([@natalisucks](https://github.com/natalisucks) / Slack: `@Natali`) |
```
I believe that removing `|` could fix the issue of moving all names into the table.